ovos_gui_plugin_shell_companion/__init__.py

```diff
@@ -37,10 +37,10 @@ def __init__(self, config: dict, bus: MessageBusClient = None,
         res_dir = join(dirname(__file__), "res")
         gui = gui or GUIInterface("ovos_gui_plugin_shell_companion",
                                   bus=bus, config=Configuration(),
-                                  resource_dir=res_dir)
-        if not gui.resource_dir:
-            LOG.info(f"Setting default GUI resource directory to: {res_dir}")
-            gui.resource_dir = res_dir
+                                  ui_directories={"qt5": res_dir})
+        if not gui.ui_directories:
+            LOG.info(f"Setting default qt5 resource directory to: {res_dir}")
+            gui.ui_directories["qt5"] = res_dir
```

Comment:
can we also add QT6 already? ui6 instead of ui folder

as discussed in https://github.com/OpenVoiceOS/ovos-shell/issues/29

or leave a TODO comment at least